### PR TITLE
Fix GTSAM MATLAB Wrapper

### PR DESCRIPTION
**Pull request by ProfFan**
This PR should

1. Update `wrap` to latest version
2. Unbreak the MATLAB wrapper (we still need to figure out a way to do templates in global function later)
3. Unbreak the CI pipeline (especially the macOS CI)

### CODE REVIEW

**Comment by ProfFan** [comment]
@varunagrawal @jlblancoc @dellaert  Could you do a quick review on this? This PR should also unbreak the macOS Python CI because of upstream changes in Homebrew.

File: wrap/CMakeLists.txt [hunk 1]
```diff
@@ -26,34 +26,15 @@ install(FILES cmake/gtwrapConfig.cmake cmake/PybindWrap.cmake
...
+# Install the gtwrap python package as a directory so it can be found for
+# wrapping.
+install(DIRECTORY gtwrap DESTINATION "${CMAKE_INSTALL_DATADIR}/gtwrap")
+
 # Install wrapping scripts as binaries to `CMAKE_INSTALL_PREFIX/bin` so they can
 # be invoked for wrapping.
-# We use DESTINATION (instead of TYPE) so we can support older CMake versions.
-install(PROGRAMS scripts/pybind_wrap.py scripts/matlab_wrap.py
-        DESTINATION ${CMAKE_INSTALL_BINDIR})
+install(PROGRAMS scripts/pybind_wrap.py scripts/matlab_wrap.py TYPE BIN)
```
Comment (varunagrawal) [inline wrap/CMakeLists.txt:36]: Should we be using TYPE? I left a comment saying that we need DESTINATION for older versions of CMake.

**Review by varunagrawal — APPROVED** [review]
LGTM except for the one comment.

File: wrap/CMakeLists.txt (hunk 1 above)
Comment (ProfFan) [reply wrap/CMakeLists.txt:36]: Agreed :) I'll push a change to this to wrap (but let's first merge this)

**Pull request merged**